### PR TITLE
Request to merge "Youtube clone" to master

### PR DESCRIPTION
**Pull request by RailonA**
We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. It has mock suggestions showing video name and views, and a mock section for commentaries. The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @RailonA  and @RailonA  . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below. 

IMPORTANT: Do not open a new Pull Request for the submission of the project. Always use the first Pull Request this will help you keep track of your changes and the TSE reviews.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

> **STATUS: CHANGES REQUIRED!**
> Hi there @RailonA and @DsunilK, congratulations on the good work done, i am impressed. But before marking the project as complete, we have to address a few issues .
> 
**The is no information from my side that can give me an idea of why the conflict with the file is, first try to format correctly the `main.css` file.**

**Later check if it has merge issues do a `git status`, see if `git` outputs any errors, see if the files that both of you have are the same as the one in the project and the one that each one of you have locally, check if github gives more explanation about the error, it usually does also gives instruction on how to fix it"**
- [ ]   Resolve the conflicting files

**The pull request message should not be a comment requesting help. For that reach out for one of the TSE via slack and explain what's the issue about.**
- [ ]   Your PR is not descriptive. Here is a suggestion
>   **We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views, and a mock section for commentaries. The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.**
> * [x]   Please remove spacing from the side of your page.
>   yt
> * [x]   Please fix the titles on the recommended videos to fit on the container.
> * [ OPTIONAL]
>   You can add a comments section too
> 
> Kindly work on the review given and submit for another review
> 
> Happy coding 
> Incase of any questions, feel free to reach me on slack (peris ndanu)
> 
> `

### [Comments]

- you can reach out on slack, send me a DM @jubaan

File: index.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    <div id="columns" class="container-fluid flex-direction ">
+
+      <section class="left-column" class="col-sm-12  col-md-8 col-lg-8">
+        <iframe class=" box-shadow" src="https://www.youtube.com/embed/lJIrF4YjHfQ" frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen>
+        </iframe>
+
+
+        <articlle>
```
Comment (descholar-ceo) [inline index.html]: Please fix this typo error, it should be `<article>` instead

File: index.html [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (descholar-ceo) [inline index.html]: Don't load scripts in your head tag, this should be done on the bottom of the body, before closing the body tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,335 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
```
Comment (descholar-ceo) [inline index.html]: Add `crossorigin` attribute to this tag for the sake of integrity validations

File: README.md [hunk 4]
```diff
@@ -1 +1,43 @@
...
+- CSS
+- Font Awesome
+
+## Live Demo
+
+[Live Demo Link](https://railona.github.io/youtube-simulator-page/)
+
+👤 **Railon Acosta**
+
+- GitHub: [@railonA](https://github.com/RailonA)
```
Comment (descholar-ceo) [inline README.md:22]: If you have more social links like `linkedin` or `twitter` I would suggest you to add them here as well

File: README.md [hunk 5]
```diff
@@ -1 +1,43 @@
-# youtube-simulator-page
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# YouTube Webpage
+> Task is to clone YouTube’s video player page using web archive link: http://archive.today/Bss88 as a source page.
+
+We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views, and a mock section for commentaries. The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.
+
+![screenshot](./File/img/demopage.png)
```
Comment (descholar-ceo) [inline README.md:8]: The current look of your project on this [Live Demo Link](https://railona.github.io/youtube-simulator-page/), doesn't match this screenshot, please update it.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @RailonA  and @DsunilK  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
### Note: Sorry that it was not mentioned earlier, but these requirements are mandatory to get your project approved

- [ ] Don't use ID selectors in your CSS file, [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png), or read [this article to learn more 
about CSS common mistakes](https://speckyboy.com/good-bad-css-practices/) So that you can avoid them.
- [ ] Make the title of your pull request descriptive and fix typo mistakes in it
- [ ] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [ ] It's good :+1: that you tried to use sementic HTML, but train yourself to use it more and more
- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Add info to under video`,  instead of `adding info to under video` refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

File: mystyle.css [hunk 6]
```diff
@@ -0,0 +1,656 @@
...
+      }
+    
+/*     
+      @media (min-width: 200px) 
+      { 
+        #sideBartext  h6{ font-size: 12px; }
+        #sidebartext-info1{ font-size: 780px; }
+        #sidebartext-info2{ font-size: 10px; }
+
+      } */
```
Comment (jubaan) [inline mystyle.css]: - [ ] Remove commented code. Only push code that is ready for production.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

> * [x] Don't use ID selectors in your CSS file, [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png), or read [this article to learn more
>   about CSS common mistakes](https://speckyboy.com/good-bad-css-practices/) So that you can avoid them.
**Edit the title of the PR to make it descriptive. There is an option to edit. DO NOT close and open a new PR**
- [ ]  Make the title of your pull request descriptive and fix typo mistakes in it
actual: `Project Compleated`
improved: `Request to merge "Apple clone" to master`

> * [x]  Implement all changes I suggested in inline comments in your codes

### [Optional]

- [ ] Remove the commented block in the `mystyle.css`. See inline

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by ElijahTobs — APPROVED** [review]
# Status: Approved! :tada: 

Congratulations! @RailonA  and @DsunilK 

### Good job implementing the changes requested by the previous TSE.

- **Before you merge, make sure you add a margin to the left side of your video player**

Happy Coding :computer: 

Feel free to reach out to me 
slack @ElijahTobs
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Pull request merged**